tests/wpt/meta/trusted-types/trusted-types-event-handlers.html.ini

```diff
@@ -1,241 +1,10 @@
 [trusted-types-event-handlers.html]
-  [Event handler onclick should be blocked.]
-    expected: FAIL
-
-  [Event handler onchange should be blocked.]
-    expected: FAIL
-
-  [Event handler onfocus should be blocked.]
-    expected: FAIL
-
-  [Event handler oNclick should be blocked.]
-    expected: FAIL
-
-  [Event handler OnClIcK should be blocked.]
-    expected: FAIL
-
-  [Event handler div.oncopy should be blocked.]
-    expected: FAIL
-
-  [Event handler div.oncut should be blocked.]
-    expected: FAIL
-
-  [Event handler div.onpaste should be blocked.]
-    expected: FAIL
-
-  [Event handler div.onauxclick should be blocked.]
-    expected: FAIL
-
-  [Event handler div.onabort should be blocked.]
-    expected: FAIL
-
-  [Event handler div.onbeforeinput should be blocked.]
-    expected: FAIL
-
-  [Event handler div.onbeforematch should be blocked.]
-    expected: FAIL
-
-  [Event handler div.onbeforetoggle should be blocked.]
-    expected: FAIL
-
-  [Event handler div.onblur should be blocked.]
-    expected: FAIL
-
-  [Event handler div.oncancel should be blocked.]
-    expected: FAIL
-
-  [Event handler div.oncanplay should be blocked.]
-    expected: FAIL
-
-  [Event handler div.oncanplaythrough should be blocked.]
-    expected: FAIL
-
-  [Event handler div.onchange should be blocked.]
-    expected: FAIL
-
-  [Event handler div.onclick should be blocked.]
-    expected: FAIL
-
-  [Event handler div.onclose should be blocked.]
-    expected: FAIL
-
-  [Event handler div.oncommand should be blocked.]
-    expected: FAIL
-
-  [Event handler div.oncontextmenu should be blocked.]
-    expected: FAIL
-
-  [Event handler div.oncontextlost should be blocked.]
-    expected: FAIL
-
-  [Event handler div.oncontextrestored should be blocked.]
-    expected: FAIL
-
-  [Event handler div.oncuechange should be blocked.]
-    expected: FAIL
-
-  [Event handler div.ondblclick should be blocked.]
-    expected: FAIL
-
-  [Event handler div.ondrag should be blocked.]
-    expected: FAIL
-
-  [Event handler div.ondragend should be blocked.]
-    expected: FAIL
-
-  [Event handler div.ondragenter should be blocked.]
-    expected: FAIL
-
   [Event handler div.ondragexit should be blocked.]
```

Comment:
It's highly likely that the ones that still fail are not implemented in Servo at all right now.